### PR TITLE
Updates keylime-init

### PR DESCRIPTION
**Pull request by geedo0**
Changes the host for stage 2 keylime.tar.gz payload and adds failover for keylime_node.

### CODE REVIEW

File: initrd/etc/keylime-init [hunk 1]
```diff
@@ -1,17 +1,25 @@
 #!/bin/sh
 # Bring up the x230's NIC, get a DHCP address and invoke keylime
 
+recovery_on_error() {
```
Comment (osresearch) [inline initrd/etc/keylime-init]: This function is now provided in `/etc/functions` in the initrd.

File: initrd/etc/keylime-init [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+}
+
 insmod /lib/modules/e1000e.ko
 udhcpc -n
 
 cd /
-wget-measure.sh 6 http://192.168.1.5/keylime.tar.gz
-tar xf keylime.tar.gz
+wget-measure.sh 6 http://keylime-server/keylime.tar.gz
+tar -xvzf keylime.tar.gz -C /
```
Comment (osresearch) [inline initrd/etc/keylime-init]: the `-C /` shouldn't be necessary if you've also done the `cd /` a few lines above.

File: initrd/etc/keylime-init (hunk 1 above)
Comment (geedo0) [inline initrd/etc/keylime-init]: Neat! I'll make sure to reuse that.

**Comment by osresearch** [comment]
Do you have any further updates after this weekend's hacking? This can be merged if it is ready.

**Comment by geedo0** [comment]
The hack we did to handle the library conflict with Keylime is not pretty so we'll have to hold off on this. Would you be opposed to either invoking ld-musl explicitly or manipulating LD_LIBRARY_PATH in this script to launch Keylime?

**Comment by osresearch** [comment]
What's the status on the PR?  Is it still relevant?

**Comment by geedo0** [comment]
I'll close this PR and resubmit when we have something end to end ready.

**Pull request closed without merging**